app/components/autocomplete/channel_mention/index.js

```diff
@@ -49,13 +50,15 @@ function mapStateToProps(state, ownProps) {
         matchTerm,
         requestStatus: state.requests.channels.getChannels.status,
         theme: getTheme(state),
+        serverVersion: state.entities.general.serverVersion || Client4.getServerVersion(),
```

Comment:
Can these ever get out of sync? We typically just use the value in state without looking at the Client4 instance directly

Reply:
Actually, since we're looking at the state first, `Client4.getServerVersion` will only get called before that state is ever set the first time which should never really happen, so that part is unnecessary. Also, now that I think of it, we shouldn't be using `Client4` here anyway since `mapStateToProps` should be pure.

Reply:
I did that by imitation I can change it in both places.

Reply:
Done, and removed from other 2 places :)